llvm/lib/Target/AIE/AIEInterBlockScheduling.cpp

```diff
@@ -353,6 +366,22 @@ bool InterBlockScheduling::updateFixPoint(BlockState &BS) {
     return false;
   }
 
+  if (MachineInstr *MINeedsHigherCap = resourcesConverged(BS);
+      InterBlockScoreboard && MINeedsHigherCap) {
+    auto Res = BS.FixPoint.PerMILatencyMargin.try_emplace(MINeedsHigherCap, 0);
```

Comment:
nit: Can we use another name for this `PerMILatencyMargin`? In feel it to tied to latency. Now it encloses a bigger scope.

Reply:
I got the same feeling, but in the end we "push" instructions up by increasing their latency to `ExitSU` so I kept the name. Do you have some name suggestion? Maybe something like `MILatencyToExitSU` would be clearer?

Reply:
I think it is fine to leave as is, or maybe something neutral, like `MIDistanceToExitSU`.